ketcher-autotests/tests/File-Management/Open-And-Save-Files/MOL-Files/mol-files.spec.ts

```diff
@@ -547,3 +549,268 @@ test('Open V3000 file with R-Groups with Fragments', async ({ page }) => {
   await openFileAndAddToCanvas('RGroup-With-Fragments.mol', page);
   await takeEditorScreenshot(page);
 });
+
+test.describe('', () => {
```

Comment:
I propose to add some meaningful string for describe

Reply:
Done